src/network/core.rs

```diff
@@ -36,6 +36,23 @@ impl Core {
         let mut response_net_addresses: Vec<NetAddress> = Vec::new();
         // interfaces map, but we only ever expect one, for response
         let mut interfaces: HashMap<String, types::NetInterface> = HashMap::new();
+        // mtu to configure, 0 means it was not set do nothing.
```

Comment:
We should not set `mtu` to 0. Although `kernel` allows it but its a bug https://serverfault.com/a/652604

So netavark will `NOOP` in case of `mtu: 0`

Reply:
I agree 0 means default so we should do nothing